pkg/new-ui/v1beta1/frontend/src/app/pages/experiment-creation/nas-operations/types.ts

```diff
@@ -0,0 +1,6 @@
+import { ParameterSpec } from 'src/app/shared/params-list/types';
+
+export interface NasOperation {
```

Comment:
Maybe we should define NAS Operation interface in k8s models also ?

Reply:
Yes, I completely removed the intermediate `types.ts` files and only rely on the `models` files